usmqe_tests/api/gluster/test_gluster_cluster.py

```diff
@@ -19,6 +20,104 @@
 ========
 """
 
+"""@pylatest api/gluster.cluster_create
+API-gluster: cluster_create
+***************************
+
+.. test_metadata:: author fbalak@redhat.com
+
+Description
+===========
+
+Positive create gluster cluster.
+"""
+
+
+def test_cluster_create_valid(
+        valid_session_credentials,
+        valid_nodes,
+        net_interface="eth0"):
+    """@pylatest api/gluster.cluster_create
+        .. test_step:: 1
+
+        Check if there is at least one gluster node for cluster creation.
+
+        .. test_result:: 1
+
+        Test passes if there is at least one gluster node.
+        """
+    api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
+    pytest.check(
+        len(valid_nodes) > 0,
```

Comment:
It checks if there are any available nodes from which you can create cluster. It will be moved as suggested by Luboš.